conf/messages.en

```diff
@@ -1017,23 +1017,23 @@ addAnotherOtherOfficial.error.required = Select yes if you want to add another o
 
 addAnotherOtherOfficial.checkYourAnswersLabel = Add another other official
 
-# Charity's Other Officials Details Summary Page Messages
+# Charity’s Other Officials Details Summary Page Messages
 # ----------------------------------------------------------
-otherOfficialsSummary.checkYourAnswers.title = Check your charity''s other officials details
-otherOfficialsSummary.checkYourAnswers.heading = Check your charity''s other officials details
-otherOfficialsSummary.checkYourAnswers.h2 = Charity''s other official
+otherOfficialsSummary.checkYourAnswers.title = Check your charity’s other officials details
+otherOfficialsSummary.checkYourAnswers.heading = Check your charity’s other officials details
+otherOfficialsSummary.checkYourAnswers.h2 = Charity’s other official
 otherOfficialsSummary.section = Officials and nominees
 
 # Section9 - Name nominees
 # ----------------------------------------------------------
 
 # Charity Nominee messages
 # ----------------------------------------------------------
-charityNominee.title = The charity''s nominee
-charityNominee.heading = The charity''s nominee
+charityNominee.title = The charity’s nominee
+charityNominee.heading = The charity’s nominee
 charityNominee.p1 = You now have the option of adding a nominee.
 charityNominee.p2 = A nominee is a person or an organisation from outside the charity who can submit Gift Aid or other tax repayment claims on your behalf. They might also receive the repayments too.
-charityNominee.p3 = Nominees aren''t ''managers'' for the purposes of the fit and proper persons test. But you do have to tell us if the charity appoints a nominee or the nominee changes.
+charityNominee.p3 = Nominees aren’t ’managers’ for the purposes of the fit and proper persons test. But you do have to tell us if the charity appoints a nominee or the nominee changes.
```

Comment:
This apostrophe is not oriented correctly, needs to be flipped
<img width="633" alt="Screenshot 2020-09-02 at 16 46 26" src="https://user-images.githubusercontent.com/28566263/92006105-3880b080-ed3c-11ea-9292-7a1bfd00a145.png">